src/genie/libs/parser/iosxe/tests/ShowIpSlaSummary/cli/equal/golden_output_1_expected.py

```diff
@@ -0,0 +1,39 @@
+expected_output = {
+    'probes': {
+            '1': {'destination': '123.23.213.32',
+                  'last_run': '21 seconds ago',
+                  'return_code': 'OK',
+                  'rtt_stats': '44',
```

Comment:
We are not entirely sure what kind of patterns we have in this field. so I would suggest to get it as is without separation.
```suggestion
                  'stats': 'RTT=44',
```

Reply:
I have changed this in the new commits to rtt_stats_mseconds (as the default is in milliseconds, however, if you would still prefer me to have it as is, I can change that again)

Reply:
what I meant by `patterns`, we might see different output instead of `RTT=number`.

For example, I found another output from cisco.com. and it just has only number. (not having `RTT=`) and we are not sure what kind of other patterns are here. so suggested to take as string as is.
https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/ipsla/command/sla-xe-3se-5700-cr-book/sla-xe-3se-3850-cr-book_chapter_010.html#wp1305239218
```
Device# show ip sla summary destination 192.0.2.2
ID       Type      Destination  State   Stats(ms)  ReturnCode  LastRun
---      ----      -----------  -----   -------  ----------  -------
100   icmp-jitter   192.0.2.2    Active   100      OK       22:49:53 PST Tue May 3 2011
101   udp-jitter    192.0.2.2    Active   100      OK       22:49:53 PST Tue May 3 2011
102   tcp-connect   192.0.2.2    Active    -      NoConnection  22:49:53 PST Tue May 3 2011
103   video         1232:232  		 Active   100      OK       22:49:53 PST Tue May 3 2011  
                      ::222                                                                  
104   video         1232:232  		 Active   100      OK       22:49:53 PST Tue May 3 2011 
                      ::222

Reply:
waiting for reply on this.